src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json

```diff
@@ -35,11 +35,11 @@
     "sulu_page.edited": "edited",
     "sulu_page.search_result": "Search Engine Result Pages Preview (SERP)",
     "sulu_page.meta_title": "Title (Meta title)",
-    "sulu_page.meta_title_info_text": "The meta title is limited to 55 characters.",
+    "sulu_page.meta_title_info_text": "The meta title is recommended to use 55 characters at most.",
```

Comment:
the limits on the tab are not hard-limits but recommendations. we should make that clear in the respective info texts 🙂